File: Iot_raspberrypi/01_gpio_digital/led_blink.py
import RPi.GPIO as GPIO
import time

LED_PIN = 4
GPIO.setmode(GPIO.BCM) #BCM pin번호를 따르겠다는 의미
GPIO.setup(LED_PIN, GPIO.OUT)

for i in range(1, 11):
    GPIO.output(LED_PIN, GPIO.HIGH)
    print("led on")
    time.sleep(0.05*i)
    GPIO.output(LED_PIN, GPIO.LOW)
    print("led off")
    time.sleep(0.05*i)

GPIO.cleanup() #pin 초기화

File: Iot_raspberrypi/01_gpio_digital/led_blink.c
#include <wiringPi.h>
#include <stdio.h>
#define LED 4

int main (void)
{
  //wiringPiSetup () ; //physical pin번호로 할때
  wiringPiSetupGpio(); //bcm pin번호로 할때
  pinMode (LED, OUTPUT) ;
  for (int i=1;i<11;i++)
  {
    digitalWrite (LED, HIGH) ; delay (50*i) ;
    printf("led on\n");
    digitalWrite (LED,  LOW) ; delay (50*i) ;
    printf("led off\n");
  }
  return 0 ; 
}